src/NuGet.Clients/NuGet.CommandLine/xlf/NuGetCommand.cs.xlf

```diff
@@ -192,8 +192,8 @@ nuget client-certs List</target>
       <trans-unit id="ConfigCommandExamples">
         <source>nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY.USER=domain\user
 nuget config HTTP_PROXY</source>
-        <target state="translated">nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY. USER=domain\user
-konfigurační HTTP_PROXY nuget</target>
+        <target state="translated">nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY.USER=domain\user
+nuget config HTTP_PROXY</target>
         <note>Please don't localize this string</note>
```

Comment:
Now that we're using Arcade's xlf tools, maybe we should update these comments so that the tooling automatically does the right thing: https://github.com/dotnet/arcade/blob/19b800616b55f3fae5098eb0dad0a63a15747a00/src/Microsoft.DotNet.XliffTasks/Model/ResxDocument.cs#L37-L38

Although if the entire string is locked, I don't understand why it's a resource in the first place 🤷

Reply:
Created issue to track it https://github.com/NuGet/Client.Engineering/issues/3417